docker-compose.yml

```
@@ -0,0 +1,6 @@
+version: '2'
+services:
+  php:
+    image: 'leroymerlinbr/php:7.3-apache'
```

No need to use apache here. Also, I think that we can move on to php 7.4

```suggestion
    image: 'leroymerlinbr/php:7.4'
```